src/analytics/task/useTaskAnalytics.ts

```diff
@@ -40,7 +40,17 @@ type Action =
   | { name: "Click Distro Link" }
   | { name: "Click Jira Summary Link" }
   | { name: "Click Build Variant Link" }
-  | { name: "Build Baron File Ticket" };
+  | { name: "Build Baron File Ticket" }
+  | { name: "Save Annotation Note" }
+  | { name: "Click Annotation Ticket Link" }
+  | { name: "Remove Annotation Issue" }
+  | { name: "Remove Annotation Suspected Issue" }
+  | { name: "Move Annotation Issue" }
+  | { name: "Move Annotation Suspected Issue" }
+  | { name: "Click Add Annotation Issue Button" }
+  | { name: "Click Add Annotation Suspected Issue Button" }
+  | { name: "Add Task Annotation Issue" }
+  | { name: "Add Task Annotation Suspected Issue" };
```

Comment:
Thse should be removed from here since they already exist in the useAnnnotationAnalytics hook